instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/patches/server.rb

```diff
@@ -26,6 +26,25 @@ def request(op, *args)
 
           private
 
+          CMD_MAX_LEN = 500
+
+          def db_statement(operation, args)
+            placeholder = "#{operation} BLOB (OMITTED)"
+            case config[:db_statement]
+            when :omit
+              command = placeholder
+            when :obfuscate
+              unsanitized_command = [operation, *args].join(' ').strip
+              command = OpenTelemetry::Common::Utilities.utf8_encode(unsanitized_command, binary: true, placeholder: placeholder)
+            end
```

Comment:
Sanitizing the statement is not the same as obfuscation.  I'd encourage you to test this by running a test and see if any of the values you set show up in the resulting span.  

Instead of taking on adding obfuscation here, I'd encourage you to simply add the `omit` and `include` options for now.

Reply:
so this logic should be under `include` flag then. Note I mostly  copy pastad from https://github.com/SomalianIvan/opentelemetry-ruby/blob/7667c3c8e969b74627c5c1afc21847e156c4d6ab/instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/utils.rb#L47-L55

Reply:
Right if it is included, then run the format code.  Otherwise do no work and do not add the `db.statement` attribute to the span.

Reply:
We can punt obfuscation to a follow up PR/issue.